community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -488,6 +488,12 @@
     @Description( "Log parameters for executed queries that took longer than the configured threshold." )
     public static final Setting<Boolean> log_queries_parameter_logging_enabled = setting( "dbms.logs.query.parameter_logging_enabled", BOOLEAN, TRUE );
 
+    @Description( "Log detailed time information for executed queries that took longer than the configured threshold." )
```

Comment:
@thobe not really.

We can't generate them because some settings in there are defined with different values than the actual defaults. With comments saying "Uncomment this line to enable XXX". And we don't want ALL settings in there.